api/src/certification/session-management/infrastructure/repositories/assessment-result-jury-comment-repository.js

```diff
@@ -0,0 +1,27 @@
+import { knex } from '../../../../../db/knex-database-connection.js';
+import { NotFoundError } from '../../../../shared/domain/errors.js';
+import { AssessmentResultJuryComment } from '../../domain/models/AssessmentResultJuryComment.js';
+
+const getLatestAssessmentResultJuryComment = async function ({ certificationCourseId }) {
+  const result = await knex('assessment-results')
+    .select('id', 'juryId', 'commentByJury')
+    .innerJoin(
+      'certification-courses-last-assessment-results',
+      'assessment-results.id',
+      'certification-courses-last-assessment-results.lastAssessmentResultId',
+    )
+    .where({ certificationCourseId })
+    .first();
+
+  if (!result) {
+    throw new NotFoundError(`Assessment result not found for certification course ${certificationCourseId}`);
+  }
+
+  return new AssessmentResultJuryComment(result);
+};
+
+const save = async function ({ id, juryId, commentByJury }) {
+  return knex('assessment-results').update({ juryId, commentByJury }).where({ id });
+};
```

Comment:
Récupérer également l'`emitter` lors de la destructuration pour le mettre à jour aussi.

```suggestion
  return knex('assessment-results').update({ juryId, commentByJury, emitter }).where({ id });
};
```

Reply:
Erratum, il ne faut pas s'occuper de l'emitter car il indique la source du scoring (pas du commentaire). 
J'ai donc retiré la notion d'emitter dans cette PR